src/Compilers/Core/Portable/Operations/Operation.cs

```diff
@@ -22,7 +22,8 @@ internal abstract class Operation : IOperation
 
         // this will be lazily initialized. this will be initialized only once
         // but once initialized, will never change
-        private IOperation _parentDoNotAccessDirectly;
+        private IOperation _parentDoNotAccessDirectly = s_sentinel;
+        private static readonly IOperation s_sentinel = new EmptyStatement(null, null, null, default, isImplicit: true); 
```

Comment:
Perhaps name `s_unset`.

Reply:
> Perhaps name s_unset.

Will rename in the next PR

---
In reply to: [163286292](https://github.com/dotnet/roslyn/pull/24358#discussion_r163286292) [](ancestors = 163286292)